standalone/nw.js

```diff
@@ -0,0 +1,74 @@
+function patchSqueakForNode(root) {
+    var fs = require("fs");
+    var Squeak = window.Squeak;
```

Comment:
It's okay that is merged it already but wanted to mention that if we could factor out references to `window` to a single point somewhere else, then running from a web browser VM or the NodeJS VM would be a matter of preference. 

I'm thinking in a project that would make use of SqueakJS as a headless backend.

cc/ @fniephaus @bertfreudenberg @ccrraaiigg @timfel